doc/build-unix.md

```diff
@@ -125,13 +125,7 @@ To build without GUI pass `--without-gui` to configure.
 
 To build with Qt 5 (recommended) you need the following:
 
-    sudo apt-get install libqt5gui5 libqt5core5a libqt5dbus5 qttools5-dev qttools5-dev-tools libprotobuf-dev protobuf-compiler
-
-If ```./configure``` fails to detect your QT installation, you have to run:
```

Comment:
migrated: why is this being removed?

Reply:
The command `sudo apt-get install libqt5* ` has conflicts and will most likely break your Qt installation.

Reply:
I think that is right. The actual list of Qt packages needed is above.